setup/nnt_environment.yml

```diff
@@ -1,100 +1,127 @@
-name: nnt
+name: nnt2
 channels:
   - defaults
-  - conda-forge
   - pytorch
+  - conda-forge
 dependencies:
```

Comment:
I would try not using the complete build-identifier as the version, as they are often not available anymore in a few weeks time.
I'd try only giving the actual version, so e.g. `ca-certificates=2021.4.13` instead of `ca-certificates=2021.4.13=haa95532_1` and then just do a quick check if the environment can still be setup.